delfin/coordination.py

```diff
@@ -350,3 +350,53 @@ def register_watcher_func(self, on_node_join, on_node_leave):
 
     def watch_group_change(self):
         self.coordinator.run_watchers()
+
+
+class GroupMembership(Coordinator):
+
+    def __init__(self, agent_id):
+        super(GroupMembership, self). \
+            __init__(agent_id=agent_id, prefix="")
+
+    def create_group(self, group):
+        try:
+            self.coordinator.create_group(group.encode()).get()
```

Comment:
Why call `get` after called `create_group`

Reply:
create_group() is async call, following get() ensure that create group is completed